security-admin/scripts/db_setup.py

```diff
@@ -205,7 +209,7 @@ def get_jisql_cmd(self, user, password ,db_name):
 				db_ssl_cert_param=" -Djavax.net.ssl.keyStore=%s -Djavax.net.ssl.keyStorePassword=%s -Djavax.net.ssl.trustStore=%s -Djavax.net.ssl.trustStorePassword=%s " %(self.javax_net_ssl_keyStore,self.javax_net_ssl_keyStorePassword,self.javax_net_ssl_trustStore,self.javax_net_ssl_trustStorePassword)
 		self.JAVA_BIN = self.JAVA_BIN.strip("'")
 		if is_unix:
-			jisql_cmd = "%s %s -cp %s:%s/jisql/lib/* org.apache.util.sql.Jisql -driver mysqlconj -cstring jdbc:mysql://%s/%s%s -u '%s' -p '%s' -noheader -trim -c \;" %(self.JAVA_BIN,db_ssl_cert_param,self.SQL_CONNECTOR_JAR,path,self.host,db_name,db_ssl_param,user,password)
+			jisql_cmd = "%s %s -cp %s:%s/jisql/lib/* org.apache.util.sql.Jisql -driver mysqlconj -cstring jdbc:mysql://%s%s/%s%s -u '%s' -p '%s' -noheader -trim -c \;" %(self.JAVA_BIN, db_ssl_cert_param, self.SQL_CONNECTOR_JAR, path, self.host, mysql_port, db_name, db_ssl_param, user, password)
```

Comment:
don't we need a colon between %s%s for jdbc:mysql://%s%s  (self.host and mysql_port)?

Reply:
ok ignore it :) i missed we are adding colon above